### ISSUE TITLE
No module named rucio.core when running rucio client commands

### ISSUE THREAD

**Issue by maxnoe** [rucio/rucio#6461]
### Description

Following the installation instructions here:
https://rucio.github.io/documentation/user/setting_up_the_rucio_client

I created a virtual environment, installed the `rucio-clients` package and just ran `rucio --help`. This resulted in the error described in #6460 .

Creating an empty configuration file at one of the search for paths now results in the error `no module named rucio.core`.

### Steps to reproduce

```
$ python3 -m venv venv
$ source venv/bin/activate
$ pip install rucio-clients
$ touch venv/etc/rucio.cfg  # work-around for #6460 
$ rucio --help
Traceback (most recent call last):
  File "/home/mnoethe/Uni/CTA/dpps/rucio-ci-test/venv/bin/rucio", line 52, in <module>
    from rucio.rse.protocols.protocol import RSEProtocol
  File "/home/mnoethe/Uni/CTA/dpps/rucio-ci-test/venv/lib/python3.12/site-packages/rucio/rse/__init__.py", line 81, in <module>
    from rucio.core.rse import get_rse_protocols, get_rse_id
ModuleNotFoundError: No module named 'rucio.core'
```

### Rucio Version

rucio-clients             33.2.0

### Additional Information

Tested both in an almalinux 9 docker container and on ubuntu 22.04

**Comment by bari12** [comment]
Is the configuration file really empty, or does it include a `[database]` section?

**Comment by maxnoe** [comment]
As you can see from the commands above, it is just `touch`ed

**Comment by bari12** [comment]
Can you check if there is another rucio.cfg in the environment:
```
    - $RUCIO_HOME/etc/
    - $VIRTUAL_ENV/etc/
    - /opt/rucio/
```
This error certainly comes due to rucio thinking it is in a server environment, which it detects by having a `database` section in the config.

**Comment by maxnoe** [comment]
I just tested in a fresh alma linux 9 container and indeed the steps above work and show the help.

However, also without touching the config file it works there, which seems #6460 was fixed in the mean time? It's still open...

Testing with rucio 34.0

**Comment by maxnoe** [comment]
This issue seems to have been fixed in rucio 33.5 and above
